tests/function/test_config.py

```diff
@@ -190,3 +190,115 @@ def test_reporter_id_in_virtwho_conf(self, virtwho, globalconf, ssh_host):
                 and result['send'] == 1
                 and result['thread'] == 1
                 and result['reporter_id'] == reporter_id)
+
+    def test_log_per_config_in_virtwho_conf(self, virtwho, globalconf, hypervisor_handler, ssh_host):
```

Comment:
In pr `https://github.com/VirtwhoQE/virtwho-test/pull/73`, I removed the `name='register_handler'` and `name='hypervisor_handler'`, which could be replaced by `name='register_data'` and `name='hypervisor_data'`.

Reply:
Updated